packages/core/react/src/WalletProvider.tsx

```diff
@@ -150,7 +143,7 @@ export function WalletProvider({
         };
     }, [adaptersWithStandardAdapters, walletName]);
     const handleConnectError = useCallback(() => {
-        if (adapter && adapter.name !== SolanaMobileWalletAdapterWalletName) {
```

Comment:
same thing here: 
`if (adapter && adapter.name !== SolanaMobileWalletAdapterWalletName && adaptersWithStandardAdapters.length === 0)`

Reply:
done!

Reply:
actually I think this one should be `if (
            adapter &&
            (adapter.name !== SolanaMobileWalletAdapterWalletName || adaptersWithStandardAdapters.length !== 0)
        ) `, let me know if you disagree

Reply:
agreed!